docs/user-guide/cli.md

```diff
@@ -12,10 +12,12 @@ npm install stylelint --save-dev
 
 ## Usage
 
-`stylelint --help` prints the CLI documentation.
+`./node_modules/stylelint/bin/stylelint.js --help` prints the CLI documentation after adding stylelint as a dependancy.
```

Comment:
```suggestion
`./node_modules/stylelint/.bin/stylelint.js --help` prints the CLI documentation after adding stylelint as a dependency.
```
I think you'll find there is a `.` period before `bin`...

Reply:
I think @ntwb meant `./node_modules/.bin/stylelint`. This is correct path, which not relies on implementation details of stylelint. `./node_modules/stylelint/bin/stylelint.js` will break if we change filename or location of this file. `./node_modules/.bin/stylelint` always will be the same.